src/gui/mainwindow.cpp

```diff
@@ -161,12 +161,12 @@ MainWindow::MainWindow(IGUIApplication *app, QWidget *parent)
     m_ui->actionExit->setIcon(UIThemeManager::instance()->getIcon(u"application-exit"_qs));
     m_ui->actionLock->setIcon(UIThemeManager::instance()->getIcon(u"object-locked"_qs));
     m_ui->actionOptions->setIcon(UIThemeManager::instance()->getIcon(u"configure"_qs));
-    m_ui->actionPause->setIcon(UIThemeManager::instance()->getIcon(u"media-playback-pause"_qs));
-    m_ui->actionPauseAll->setIcon(UIThemeManager::instance()->getIcon(u"media-playback-pause"_qs));
-    m_ui->actionStart->setIcon(UIThemeManager::instance()->getIcon(u"media-playback-start"_qs));
-    m_ui->actionStartAll->setIcon(UIThemeManager::instance()->getIcon(u"media-playback-start"_qs));
+    m_ui->actionPause->setIcon(UIThemeManager::instance()->getIcon(u"torrent-stop"_qs));
+    m_ui->actionPauseAll->setIcon(UIThemeManager::instance()->getIcon(u"torrent-stop"_qs));
+    m_ui->actionStart->setIcon(UIThemeManager::instance()->getIcon(u"torrent-start"_qs));
+    m_ui->actionStartAll->setIcon(UIThemeManager::instance()->getIcon(u"torrent-start"_qs));
     m_ui->menuAutoShutdownOnDownloadsCompletion->setIcon(UIThemeManager::instance()->getIcon(u"task-complete"_qs));
-    m_ui->actionManageCookies->setIcon(UIThemeManager::instance()->getIcon(u"preferences-web-browser-cookies"_qs));
+    m_ui->actionManageCookies->setIcon(UIThemeManager::instance()->getIcon(u"preferences-cookies"_qs));
```

Comment:
Would you consider `http-cookies` (or `browser-cookies`)?

Reply:
I tried to use `preference-*` for icons in the settings. Do you still want `browser-cookies`?

Reply:
It depends on the meaning you wish to convey on the icons.

`http-cookies` is clear itself.
`preferences-cookies` shows that it has relations (in an unknown way) with the "preferences". If you think this relation is significant and prefer it then I would suggest using `preferences-http-cookies` since just `cookie` is too vague.

Reply:
Then let's go with, `browser-cookies`, by name it is the best option.

Reply:
Done.